AppLovin MAX Demo App - ObjC/AppLovin MAX Demo App - ObjC/MAX/Native Ads/ALMAXAdPlacerCollectionViewController.m

```diff
@@ -0,0 +1,61 @@
+//
+//  ALMAXAdPlacerCollectionViewController.m
+//  AppLovin MAX Demo App - ObjC
+//
+//  Created by Ritam Sarmah on 4/1/22.
+//  Copyright © 2022 AppLovin Corporation. All rights reserved.
+//
+
+#import "ALMAXAdPlacerCollectionViewController.h"
+#import "ALTextCollectionViewCell.h"
+#import <AppLovinSDK/AppLovinSDK.h>
+
+@interface ALMAXAdPlacerCollectionViewController()<MAAdPlacerDelegate>
+
+@property (nonatomic, strong) MACollectionViewAdPlacer *adPlacer;
+@property (nonatomic, strong) NSArray<NSString *> *data;
+
+@end
+
+@implementation ALMAXAdPlacerCollectionViewController
+
+- (void)viewDidLoad
+{
+    [super viewDidLoad];
+    
+    self.data = [UIFont.familyNames sortedArrayUsingSelector: @selector(compare:)];
+    
+    MAAdPlacerSettings *settings = [MAAdPlacerSettings settingsWithAdUnitIdentifier: @"YOUR_AD_UNIT_ID"];
+    [settings addFixedPosition: [NSIndexPath indexPathForItem: 2 inSection: 0]];
+    [settings addFixedPosition: [NSIndexPath indexPathForItem: 8 inSection: 0]];
+    settings.repeatingInterval = 5;
+    
```

Comment:
Maybe we can add a note here or something about the APIs that can be used to bind custom native ad views for custom integration.